GrandCentralBoard/App/ViewControllers/LoadingViewController.swift

```diff
@@ -5,20 +5,33 @@
 
 import UIKit
 import GrandCentralBoardCore
+import Alamofire
 
-private let dataDownloader = DataDownloader()
-private let availableBuilders: [WidgetBuilding] = [WatchWidgetBuilder(dataDownloader: dataDownloader), BonusWidgetBuilder(dataDownloader: dataDownloader)]
-private let useLocal = debugBuild
+
+private let debugBuild: Bool = _isDebugAssertConfiguration()
+
+// Set to `true` if you want to load bundled `configuration.json`.
+private let shouldUseLocalConfigurationOnDebugBuild = false
```

Comment:
Well it's great solution, sure. It will be rarely used. I would add an alert with two options if it was to be used frequently.